apps/frontend/src/libs/modules/notification/libs/enums/default-message.enum.ts

```diff
@@ -0,0 +1,8 @@
+const DefaulteMessage = {
+	ERROR: "An error occurred.",
+	INFO: "Info.",
+	SUCCESS: "Operation completed successfully.",
+	WARN: "Warn.",
+};
```

Comment:
```suggestion
const DEFAULT_MESSAGE = "Unexpected error."
```

Reply:
Should I use default message "Unexpected error." even in info and success methods? I thought this message is not appropriate for info and success methods.

Reply:
When we display success message, the message will be explicitly specified. The middleware will handle only error case, meaning that the default message will only be used for error case